### ISSUE TITLE
Strange behavior of Plone Site when running bin/test

### ISSUE THREAD

**Issue by idgserpro** [collective/collective.cover#586]
We're trying to debug a test but the Plone Site from `bin/instance` is different from `bin/test` Plone Site.

How to test: run bin/instance in a bootstrapped collective.cover. Go to the ZMI and portalquickinstaller. You should see something similar to this image:

![selecao_124](https://cloud.githubusercontent.com/assets/8203476/13155865/f984bef8-d666-11e5-9d78-eb3ab44f6dcd.png)

Now edit a robot test, adding `Sleep  10 minutes` or so. Go to the ZMI in the new http://localhost:55001/ Plone Site created for robots. Go to the ZMI and portalquickinstaller. You'll see:

![selecao_125](https://cloud.githubusercontent.com/assets/8203476/13155896/24978756-d667-11e5-9ec9-e292d890e430.png)

If you go to a bunch of tools (or error_log), you always get redirected to

> There are no transactions that can be undone. 

and if you try to go into any created content, you get an error.

We have tested this in other packages (like collective.messagesviewlet) and the package doesn't have this Plone Site difference behavior when using `bin/test`. What from collective.cover is causing this difference? This problem is holding us back in a test.

**Comment by hvelarde** [comment]
my guess is this line: https://github.com/collective/collective.cover/blob/master/src/collective/cover/configure.zcml#L15

we should replace that by an explicit list.

on the other side, collective.cover uses Grok and all the magic included with it; we should get rid of that also.

**Comment by idgserpro** [comment]
`bin/instance` and bin/test uses the same configure.zcml, but only in `bin/test` we have this erratic behavior. Do you think there's some magic happening there that only applies to `bin/test`? This is really hard to debug. :( Do you know if there's anyone from Plone Framework team that could help us in this manner? Maybe other packages have the same problem we're facing right now.

**Comment by hvelarde** [comment]
yes, they use the same `configure.zcml` but it behaves in a different way: the line I pointed out is relevant because tests [explicitly disable autoinclude feature set](https://pypi.python.org/pypi/plone.app.testing/4.2.5#layer-reference).

ask on #plone to see if anybody has experienced issues like that.
